### PR TITLE
Added example in the readme

### PR DESCRIPTION
**Pull request by btpoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,42 @@ npm install --save react-native-dimmed-view
 
 ## Example
 
-Example coming soon.
+```
```
Comment (maluramichael) [inline README.md]: Add javascript after the ``` so the code is highlighted

**Comment by maluramichael** [comment]
Nice. Thanks @btpoe. Fix #4

**Comment by maluramichael** [comment]
I accidentally clicked on close PR so please don't mind.

**Comment by btpoe** [comment]
no worries.

**Pull request merged**